cluster-autoscaler/core/utils.go

```diff
@@ -411,10 +411,20 @@ func removeOldUnregisteredNodes(unregisteredNodes []clusterstate.UnregisteredNod
 			}
 			err = nodeGroup.DeleteNodes([]*apiv1.Node{unregisteredNode.Node})
 			if err != nil {
-				klog.Warningf("Failed to remove node %s: %v", unregisteredNode.Node.Name, err)
-				logRecorder.Eventf(apiv1.EventTypeWarning, "DeleteUnregisteredFailed",
-					"Failed to remove node %s: %v", unregisteredNode.Node.Name, err)
-				return removedAny, err
+				_, wasPlaceholder := err.(*cloudprovider.PlaceholderDeleteError)
+				// this means only a placeholder instance was deleted - it is an instance, that was requested,
+				// but was not create before StartUpTimeout. It means something's wrong with this specific
+				// node group and we temporarily suspend requesting new instances from it by registering
+				// a failed scale up
+				if wasPlaceholder {
+					klog.Warningf("Timeout trying to scale node group %s, enabling backoff for the group", nodeGroup.Id())
+					clusterStateRegistry.RegisterFailedScaleUp(nodeGroup, metrics.Timeout, time.Now())
```

Comment:
The failed scale-up logic should happen regardless, triggered from https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/clusterstate/clusterstate.go#L237. At minimum this will mess up metrics (you'll double count failed scale-ups), not sure if it breaks anything else.